bot/cogs/grades_data/grades_cog.py

```diff
@@ -150,8 +150,18 @@ def initialize(self):
                     self.master_prof_list = json.load(f)
             
         else:
-            log.info('FILES NOT FOUND')
+            notFound = ''
+            
+            if not os.path.isfile('bot/cogs/grades_data/assets/master.json'): 
+                notFound += 'master.json '
+            
+            if not os.path.isfile('bot/cogs/grades_data/assets/master_prof.json'):
+                notFound += 'master_prof.json'
 
+            log.error(f'{notfound} file not found, aborting grades command')
```

Comment:
whoops, found needs to be capsed, while we are at it lets just change the variable to snake case as is convention defined by pep8 so ```not_found```